content/installation/administrator/admin_install_rtools/index.md

```diff
@@ -10,19 +10,19 @@ tags: ["r", "windows", "installation"]
 
 ### Installatie en upgrade
 
-De installatiebestanden zijn beschikbaar via https://cloud.r-project.org/bin/windows/Rtools/
+De installatiebestanden zijn beschikbaar via [cloud.r-project.org](https://cloud.r-project.org/bin/windows/Rtools/).
+Kies de 64-bit versie van de installer.
 
 Bij een upgrade dient eerst de vorige versie te worden verwijderd.
```

Comment:
This is not strictly necessary.  I have both installed. As long as the path variable points to the correct one.